docs/ide-integrations/visual-studio/Visual-Studio-Integration-Navigation-Features.md

```diff
@@ -0,0 +1,26 @@
+
+# Navigation Features
+
+You can navigate between the methods in your bindings and the associated
+steps in your Gherkin feature files.
+
+## Navigating from a Binding to Steps in Gherkin Files
```

Comment:
Many people don't know the `Binding` term, I would use "step definition" throughout this section and the next one.